Tests/UIKitTests/UIImageViewExtensionsTests.swift

```diff
@@ -54,6 +54,16 @@ final class UIImageViewExtensionsTests: XCTestCase {
         let blurredImageView = imageView.blurred(withStyle: .extraLight)
         XCTAssertEqual(blurredImageView, imageView)
     }
+    
+    func testRemoveBlur() {
+        let imageView = UIImageView()
```

Comment:
Once you move this to `UIViewExtensions`
```suggestion
        let imageView = UIView()
```

Reply:
I have completed the changes.